PINCE.py

```diff
@@ -1787,13 +1789,15 @@ def __init__(self, parent, description=tr.NO_DESCRIPTION, address="0x", value_ty
         guiutils.fill_endianness_combobox(self.comboBox_Endianness, vt.endian)
         self.lineEdit_Description.setText(description)
         self.offsetsList = []
-        if not isinstance(address, typedefs.PointerType):
+        self.offsetDerefLabels = []
```

Comment:
From here until line 1927 are the only changes made in Part 2 UI in PINCE.py.

You can ignore everything in libpince/ as those are from Part 1.